### PR TITLE
tidb-server: Add bootstrap methord to create. user table.

### PR DESCRIPTION
**Pull request by shenli**
1. Create mysql.user table and add default user.
2. Change tidb-server unittest port to 4001 to prevent conflict with
tidb-server default port.

@coocood

### CODE REVIEW

**Comment by coocood** [comment]
LGTM

File: tidb-server/server/driver_tidb.go [hunk 1]
```diff
@@ -274,13 +275,39 @@ func convertColumnInfo(fld *field.ResultField) (ci *ColumnInfo) {
 	return
 }
 
-// CreateTiDBTestDatabase creates test and gotest database for test usage.
-func CreateTiDBTestDatabase(store kv.Storage) {
+// Bootstrap initiate TiDB server
```
Comment (qiuyesuifeng) [inline tidb-server/server/driver_tidb.go]: Comment end of .

**Comment by shenli** [comment]
PTAL

**Comment by shenli** [comment]
PTAL

**Comment by ngaut** [comment]
LGTM

**Pull request merged**